src/Authentication/Configuration/PaginationOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Altinn.Platform.Authentication.Configuration;
+
+/// <summary>
+/// The pagination options
+/// </summary>
+public class PaginationOptions
```

Comment:
Isn't this change already merged?

Reply:
yes, this PR should come after PR https://github.com/Altinn/altinn-authentication/pull/767